label_studio_ml/tests/test_basic.py

```diff
@@ -55,16 +55,3 @@ def test_webhook_predict_simple_text_classifier():
 @pytest.mark.skipif(ML_BACKEND != "simple_text_classifier", reason="Test for simple_text_classifier")
 def test_train_simple_text_classifier():
     pass
-
-
-@pytest.mark.second
-@pytest.mark.skipif(ML_BACKEND != "the_simplest_backend", reason="Test for simple_text_classifier")
-def test_setup_the_simplest_backend():
```

Comment:
Could you please revert test deletion?

Reply:
From my reading it is a duplicate of lines 30-40, I cannot see how this would run as it redefines an already-defined name. Are you saying this is deliberate? https://github.com/heartexlabs/label-studio-ml-backend/blob/b2d50f78df78d1d265f6bad6cf4b8ca64e1f313d/label_studio_ml/tests/test_basic.py#L30-L40

Also note that the `reason` gives the description "Test for simple_text_classifier" while both the condition argument in the decorator and the funcdef name itself both refer to `the_simplest_backend`, which is a different example.

Happy to delete just wanted to confirm you've reviewed what I tried to highlight by deleting